src/helpers/inlineStylesHandler.js

```diff
@@ -0,0 +1,41 @@
+import { forEach } from "./forEach";
+import { getCamelCase } from "./getCamelCase";
+
+function editInlineStyles(el, sheet, action) {
+    let selectors = Object.keys(sheet && sheet.rulesMap);
+    if (!selectors || selectors.length === 0) {
+      return;
+    }
+    var doc;
+    if (el.tagName === 'SLOT') {
```

Comment:
you should not check el.tagname and instead check if the el lies in a shadow dom then gets its shadow root